### PR TITLE
Remove Event-Bus sources

### PR DESCRIPTION
**Pull request by radufa**
**Description**

Changes proposed in this pull request:

- Remove "event-bus" component
- Update "event-service"component
- Update Installation yaml files
- Update Application Operator yaml files

**Related issue(s)**

kyma-project/kyma#7730

### CODE REVIEW

File: components/application-operator/charts/application/templates/deployment.yaml [hunk 1]
```diff
@@ -97,8 +97,6 @@ spec:
         - "/eventservice"
         - "--externalAPIPort={{ .Values.eventService.deployment.args.externalAPIPort }}"
         - "--eventMeshURL=http://{{ .Release.Name }}.{{ .Release.Namespace }}"
-        - "--eventsTargetURLV1={{ .Values.eventService.deployment.args.eventsTargetURLV1 }}"
```
Comment (Abd4llA) [inline components/application-operator/charts/application/templates/deployment.yaml:100]: as far as I can see here https://github.com/kyma-project/kyma/blob/master/components/event-service/cmd/eventservice/options.go#L21 this cmd arg is still valid and used, and if it's not sent, the default is the event mesh new endpoint . am I missing something?

File: resources/core/charts/kubeless/values.yaml [hunk 2]
```diff
@@ -15,4 +15,4 @@ config:
     secret: ""
 
 tests:
-  enabled: true
+  enabled: false
```
Comment (Abd4llA) [inline resources/core/charts/kubeless/values.yaml:18]: is there a reason for disabling the tests?

File: components/application-operator/charts/application/templates/deployment.yaml (hunk 1 above)
Comment (k15r) [reply components/application-operator/charts/application/templates/deployment.yaml:100]: yeah, during refactoring eventmesh we might have forgotten this one
i will remove it

File: resources/core/charts/kubeless/values.yaml (hunk 2 above)
Comment (k15r) [reply resources/core/charts/kubeless/values.yaml:18]: yes:

Comment (k15r) [reply resources/core/charts/kubeless/values.yaml:18]: This is the PR i am talking about: https://github.com/kyma-project/kyma/pull/7800

**Pull request merged**